samples/README.md

```diff
@@ -1,204 +1,67 @@
-  automlTranslationDataset.js list-datasets   list all Datasets
-  automlTranslationDataset.js get-dataset     Get a Dataset
-  automlTranslationDataset.js delete-dataset  Delete a dataset
-  automlTranslationDataset.js import-data     Import labeled items into dataset
-
-Options:
-  --version            Show version number                                                                     [boolean]
-  --computeRegion, -c  region name e.g. "us-central1"                                  [string] [default: "us-central1"]
-  --datasetName, -n    Name of the Dataset                                             [string] [default: "testDataSet"]
-  --datasetId, -i      Id of the dataset                                                                        [string]
-  --filter, -f        Name of the Dataset to search for              [string] [default: "translationDatasetMetadata:*"]
-  --multilabel, -m     Type of the classification problem, False - MULTICLASS, True - MULTILABEL.
-                                                                                               [string] [default: false]
-  --outputUri, -o      URI (or local path) to export dataset                                                    [string]
-  --path, -p           URI or local path to input .csv, or array of .csv paths
-                                                            [string] [default: "gs://nodejs-docs-samples-vcm/en-ja.csv"]
-  --projectId, -z      The GCLOUD_PROJECT string, e.g. "my-gcloud-project"     [number] [default: "nodejs-docs-samples"]
-  --source, -s         The source language to be translated from                                                [string]
-  --target, -t         The target language to be translated to                                                  [string]
-  --help               Show help                                                                               [boolean]
-
-Examples:
-  node automlTranslationDataset.js create-dataset -n "newDataSet" -s "en" -t "ja"
-  node automlTranslationDataset.js list-datasets -f "translationDatasetMetadata:*"
-  node automlTranslationDataset.js get-dataset -i "DATASETID"
-  node automlTranslationDataset.js delete-dataset -i "DATASETID"
-  node automlTranslationDataset.js import-data -i "dataSetId" -p "gs://myproject/mytraindata.csv"
-
-For more information, see https://cloud.google.com/translate/docs
-```
-
-[automlTranslationDataset_docs]: https://cloud.google.com/translate/docs
-[automlTranslationDataset_code]: automl/automlTranslationDataset.js
-
-[shell_img]: //gstatic.com/cloudssh/images/open-btn.png
-[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-translate&page=editor&open_in_editor=samples/README.md
+[![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-translate&page=editor&open_in_editor=samples/quickstart.js,samples/README.md)
```

Comment:
Meep.  It's outside of the scope of this change, but I hate to give this all up.  Could you file an issue to come back around and do the sample split with metadata so we keep a nicer README here?

Reply:
Yes, we will open a couple issues related to:

1. adding `body`/`samples_body` to TOC.
2. getting `sample-metadata` added, and figuring out if we're going to show the v3 samples in the README.md.

Reply:
1. https://github.com/googleapis/synthtool/issues/247